src/explorer/databaseTreeItem.ts

```diff
@@ -59,7 +59,7 @@ export default class DatabaseTreeItem extends vscode.TreeItem
     return element;
   }
 
-  listCollections(): Promise<string[]> {
```

Comment:
Worth mentioning that while this can be improved, e.g. with a dedicated type, for the time being, I just made it more correct as `dataService.listCollections()` returns a list of objects, not a list of strings.